setup.py

```diff
@@ -51,10 +51,15 @@
         "Topic :: Scientific/Engineering",
         "Topic :: Scientific/Engineering :: Medical Science Apps.",
         "Topic :: Scientific/Engineering :: Information Analysis",
-        "Topic :: Software Development :: Libraries"
+        "Topic :: Software Development :: Libraries",
+        "Operating System :: Android:",
+        "Operating System :: Microsoft :: Windows",
+        "Operating System :: POSIX",
+        "Operating System :: Unix",
+        "Operating System :: MacOS"
```

Comment:
Not until CMake is more pervasive in the Python ecosystem ;-). Current classifiers are here:

  https://pypi.python.org/pypi?%3Aaction=list_classifiers